firewood/tests/db.rs

```diff
@@ -99,11 +101,18 @@ fn test_revisions() {
             .collect();
         key
     };
+
+    let mut tmpdir: PathBuf = std::env::var_os("CARGO_TARGET_DIRY")
```

Comment:
Same type... but are you actually trying to stuff databases in `CARGO_TARGET_DIR` instead of `CARGO_TARGET_TMPDIR`?